gamefixes-steam/256330.py

```diff
@@ -0,0 +1,15 @@
+""" WRC 4
+"""
+#pylint: disable=C0103
+
+from protonfixes import util
+
+def main():
+    #Black screen without it
+    util.protontricks('d3dx9_42')
+    util.protontricks('d3dx9_43')
+    util.protontricks('faudio')
```

Comment:
Why does this solve? Using the faudio verb will install downgraded faudio DLLs and use them. Are you sure this is necessary?

Reply:
Tried without it, looks like it works now, without xact, audio sliders do not work in game. I also forgot to add wmp11 to the script to make background videos work.